### ISSUE TITLE
Up/down buttons used in tables lack sufficient colour contrast

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#4499]
The up/down buttons used to help users sort tables lack sufficient contrast with the background. This should be addressed in all repos that share similar styling for these buttons.

<img width="500" height="512" alt="Image" src="https://github.com/user-attachments/assets/44595e18-0d6f-4c37-827c-ae3ceb0d42db" />
